snippets/shorten.md

```diff
@@ -0,0 +1,23 @@
+---
+title:  shorten
```

Comment:
Remove the additional space before the title here.

Reply:
Completed.